### PR TITLE
Final step

### PR DESCRIPTION
**Pull request by muja-b**

### CODE REVIEW

**Comment by sudqijawabreh** [comment]
didn't we have two projects one for the code and one for the tests?  Where is the project for the code ?

**Comment by muja-b** [comment]
its in the using.cs 
vs code wont detect the code otherwise

File: Kata/Calculator.Tests/Usings.cs [hunk 1]
```diff
@@ -13,12 +13,18 @@ class mainClass
...
-        if(numbers.Length==0)return 0;
+        if(numbers.Equals(""))return 0;
         var mynums=numbers.Split(',');
-        if(mynums.Length==1)return Int32.Parse(mynums[0]);
-        int number1=Int32.Parse(mynums[0]);
-        int number2=Int32.Parse(mynums[1]);
-        return number1+number2;
-     }   
+        var myint=Parse(mynums);
+        var mysum=myint.Aggregate((sum,x)=>sum+=x);
```
Comment (sudqijawabreh) [inline Kata/Calculator.Tests/Usings.cs]: linq .Sum

File: Kata/Calculator.Tests/Usings.cs [hunk 2]
```diff
@@ -13,12 +13,18 @@ class mainClass
     class calculator
     {
      public int add(string numbers){
-        if(numbers.Length==0)return 0;
+        if(numbers.Equals(""))return 0;
```
Comment (sudqijawabreh) [inline Kata/Calculator.Tests/Usings.cs:16]: .IsNullOrEmpty

File: Kata/Calculator.Tests/Usings.cs [hunk 3]
```diff
@@ -22,10 +22,22 @@ class calculator
...
         var myint=Parse(mynums);
+        var negitaveObj=checkNegitaveNumbers(myint);
+        if(negitaveObj.negitaveExists){
+            throw new NegitaveNotAllowedException(negitaveObj.negitaves);
+        }
         var mysum=myint.Aggregate((sum,x)=>sum+=x);
         return mysum;
      }
 
+        private negitaveList checkNegitaveNumbers(List<int> myint)
```
Comment (sudqijawabreh) [inline Kata/Calculator.Tests/Usings.cs:51]: Get list of negative numbers could done the same

**Pull request merged**